x-pack/plugins/uptime/public/components/monitor/synthetics/executed_step.tsx

```diff
@@ -26,24 +27,9 @@ export const ExecutedStep: FC<ExecutedStepProps> = ({ step, index, checkGroup })
   return (
     <>
       <div style={{ padding: '8px' }}>
-        <div>
-          {step.synthetics?.step?.index && checkGroup ? (
-            <StepDetailLink checkGroupId={checkGroup} stepIndex={step.synthetics.step.index!}>
-              <EuiText>
-                <strong>
-                  <FormattedMessage
-                    id="xpack.uptime.synthetics.executedStep.stepName"
-                    defaultMessage="{stepNumber}. {stepName}"
-                    values={{
-                      stepNumber: index + 1,
-                      stepName: step.synthetics?.step?.name,
-                    }}
-                  />
-                </strong>
-              </EuiText>
-            </StepDetailLink>
-          ) : (
-            <EuiText>
+        <EuiFlexGroup>
+          <EuiFlexItem grow={false} style={{ flexBasis: 325 }}>
```

Comment:
Could we make a `style` version of this so we don't have the inline style in the middle of the component tree?

Reply:
done